### ISSUE TITLE
RTH not working 

### ISSUE THREAD

**Issue by 1ALeRT2** [ImpulseRC/OSD#89]
Hello.  I have wolf v3 pdb + kiss v2 fc + matek m8Q.  Configured wolf v3 pdb - GPS coordinates, speed, altitude, distance to home are shown correctly in OSD.  In addition, the pdb settings are as follows: 3d speed is off, the altitude reading from the home position, in the gps settings 10KHz, the magnetometer is calibrated and shows correctly in flight, a switch for rescue (RTH) is assigned to fc in the high position, to the same switch is assigned to align in  position middle.  Return to home does not work when RTH is turned on.  What could be the problem?

**Comment by awolf78** [comment]
What firmware do you run on the KISS FC? Did you configure a GPS in the KISS GUI (should not be configured if connected to the Wolf)?

**Comment by 1ALeRT2** [comment]
Firmware 1.3p-rc46f.

**Comment by 1ALeRT2** [comment]
does it mean that i don't need to configure the rth switch in the gui?

**Comment by 1ALeRT2** [comment]
what then needs to be configured in kiss gui?

**Comment by 1ALeRT2** [comment]
I understand there is no solution to the problem with gps?

**Comment by awolf78** [comment]
Yes there is, but it requires a firmware update. Cannot make any promises when I can fix it, but I will.

**Comment by 1ALeRT2** [comment]
I will wait

**Comment by awolf78** [comment]
Live now
